### DISCUSSION TITLE
Retrieve indexOnly details from JanusGraph index

### DISCUSSION THREAD

**Discussion by ntrhieu89** [JanusGraph/janusgraph#2778] [Q&A] [answered]
I posted the question here in Stackoverflow https://stackoverflow.com/questions/68855212/retrieve-indexonly-from-janusgraph. However, I believe currently it is not doable (to retrieve the details of indexOnly from JanusGraph index). Please correct me if I am wrong.

Could you expose this detail from JanusGraph API?
It is useful in my case where I want to re-create the graph schema of an existing setup to another setup.

**Comment by vtslab** [comment, marked as answer]
Good news, this got fixed in the upcoming v0.6.0 release of JanusGraph (already available as prerelease at [https://github.com/JanusGraph/janusgraph/releases/tag/v0.6.0](https://github.com/JanusGraph/janusgraph/releases/tag/v0.6.0)).

See the mgmt.getIndexOnlyConstraint(indexName) method at: [https://github.com/JanusGraph/janusgraph/blob/v0.6.0/janusgraph-core/src/main/java/org/janusgraph/core/schema/JanusGraphManagement.java](https://github.com/JanusGraph/janusgraph/blob/v0.6.0/janusgraph-core/src/main/java/org/janusgraph/core/schema/JanusGraphManagement.java)
